components/watermark/index.en-US.md

```diff
@@ -49,3 +49,20 @@ Add specific text or patterns to the page.
 | fontWeight | font weight | `normal` \| `light` \| `weight` \| number | normal |  |
 | fontFamily | font family | string | sans-serif |  |
 | fontStyle | font style  | `none` \| `normal` \| `italic` \| `oblique` | normal |  |
+
+## FAQ
+
+### Handle abnormal image watermarks
+
+When using an image watermark and the image loads abnormally, you can add `content` at the same time to prevent the watermark from becoming invalid.
```

Comment:
```suggestion
When using an image watermark and the image loads abnormally, you can add `content` at the same time to prevent the watermark from becoming invalid.(since 5.2.3)
```